luarules/gadgets/unit_respawning.lua

```diff
@@ -161,36 +180,39 @@ if gadgetHandler:IsSyncedCode() then
 			}
 
 			if respawnMetaList[unitID].effigy ~= "none" then
-				
-				for i = 1, 1000, 100 do
-					local x,y,z = spGetUnitPosition(unitID)
-					local blockType, blockID = Spring.GetGroundBlocked(x+i, z+i)
-					local groundH = Spring.GetGroundHeight(x+i, z+i)
-
+				local blockedIncriment = 1
```

Comment:
```suggestion
				local blockedIncrement = 1
```

Reply:
Mispelling. Had to change it in a few other locations too. You'll see such in a recent commit.